t/basic.t

```diff
@@ -18,7 +18,8 @@ if ($] >= 5.010000) {
 END
 }
 
-if ($] >= 5.010001) {
+use warnings ();
+if ($] >= 5.010001 && exists $warnings::Offsets{"experimental::smartmatch"}) {
```

Comment:
This would make the test not run on perl < 5.18

Reply:
Ah sorry, I was changing the patch at the last minute and forgot the negation. Will propose another.